models/tax-category/src/tax-category-draft/presets/index.ts

```diff
@@ -1,4 +1,4 @@
-import * as SampleDataFashion from './sample-data-fashion';
+import SampleDataFashion from './sample-data-fashion';
```

Comment:
Nicola suggested (a nit) that this preset should be lowercased for consistency

Reply:
I like it

Reply:
[6217c2c](https://github.com/commercetools/test-data/pull/208/commits/6217c2c518fae17efc6ee3217d0f2437299b8740)